eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -116,24 +107,24 @@ $responses = @{}
 
 LogInfo "Processing PackageInfo at $configFileDir"
 
-$packageProperties = Get-ChildItem -Recurse -Force "$configFileDir" `
-  | Where-Object { 
+$packageInfoFiles = Get-ChildItem -Recurse -Force "$configFileDir" `
```

Comment:
Do we want to process all package infos here or just the selected set?

Reply:
As per our chat, it always processes everything.